Tests/RespondersTests/RouteResponderTest.swift

```diff
@@ -98,7 +98,7 @@ class RouteResponderTest: XCTestCase {
 
       let expected = "GET /someRoute HTTP/1.1"
 
-      XCTAssertEqual(response.body!, expected.toBytes)
+      XCTAssertEqual(response.body!.toBytes, expected.toBytes)
```

Comment:
yeah, the test would fail since the logs wouldn't be updated on an invalid request